openwisp_users/base/models.py

```diff
@@ -206,6 +206,22 @@ def __str__(self):
     class Meta:
         abstract = True
 
+    def add_user(self, user, is_admin=False, **kwargs):
+        """
+        We override this method from the upstream dependency to
+        skip the creation of the organization owner, which we perform
+        automatically via a signal receiver.
+        Without this change, the add_user method would throw IntegrityError.
+        """
+
+        if self.users.all().exists():
```

Comment:
Are you sure of this?
Wasn't the core flagging the user as admin if there were no user for the org?
While here it's flagging the user as admin if there's already users for the org.

Shouldn't it be `if not self.users.all().exists()`?

Reply:
@nemesifier really sorry for this silly mistake. Have add not in the if statement.